requirements.txt

```diff
@@ -8,7 +8,6 @@ psutil==5.9.8
 pynvml==11.5.0
 rouge==1.0.1
 sentencepiece==0.2.0
-torch==2.0.0
```

Comment:
Why was this removed?

Reply:
it is already stated in the repo for people to install their own version of torch. e.g for me now. I only use torch 2.0.0 on the Intron single GPU machine.